addons/bindings.md

```diff
@@ -27,13 +27,36 @@ The following bindings can be installed and configured directly from within Pape
 {% for binding in sorted_bindings %}| [{{ binding.label }}]({{base}}/addons/bindings/{{ binding.id }}/readme.html) | {{ binding.description }} |
 {% endfor %}
 
-## Legacy 1.x Bindings
+## 1.x Bindings
 
 Bindings developed for the first version of openHAB can also be used with openHAB 2.
 Please be aware, that these bindings function and behave differently.
+They do for example not support the auto-discovery feature of openHAB 2 and do not define [openHAB 2 things and channels]({{base}}/concepts/things.html).
 For beginners it is recommended to resort to native bindings if possible.
 
-To be able to see and install 1.x bindings needs to be activated in the system settings.
+If there is already a openHAB 2 version of a binding available, old bindings are referred to as "Legacy 1.x Bindings".
+To be able to see and install legacy 1.x bindings, one has to activate "Include Legacy 1.x Bindings" through either Paper UI or `addons.cfg`.
+
+Configuration and usage of 1.x bindings is best done in configuration files.
```

Comment:
> best done

only true in the case where the 1.x add-on does not yet have an `ESH-INF/binding/binding.xml` or `ESH-INF/config/config.xml` file.  If it does, then the user has a choice of configuring via Paper UI, .cfg file, or if they really want to, the Karaf console `config` commands.